package.json

```diff
@@ -1,27 +1,13 @@
 {
-  "name": "cached-proxy",
-  "version": "1.0.0",
-  "main": "index.js",
-  "repository": "git@github.com:unifiprotocol/cached-proxy.git",
-  "author": "info@unifiprotocol.com",
-  "license": "MIT",
-  "scripts": {
-    "build": "tsc --incremental",
-    "tsc": "tsc --incremental",
-    "start": "node build/app.js"
-  },
-  "dependencies": {
-    "apicache": "^1.6.3",
-    "cors": "^2.8.5",
-    "express": "^4.17.2",
-    "http-proxy-middleware": "^2.0.2",
-    "morgan": "^1.10.0",
-    "typescript": "^4.5.5"
-  },
-  "devDependencies": {
-    "@types/apicache": "^1.6.1",
-    "@types/cors": "^2.8.12",
-    "@types/express": "^4.17.13",
-    "@types/morgan": "^1.9.3"
-  }
+	"dependencies": {
```

Comment:
Could you make come back the rest of the info?
We need the scripts to make it work in the CI/CD/Deployment